### ISSUE TITLE
Confounded Seasons and Regions

### ISSUE THREAD

**Issue by larogers123** [mpinsky/proj_ranges#5]
We currently deal with seasons and regions inconsistently. For instance, the NEUS is divided into two "regions", i.e. surveys, which represent the two seasons surveys are conducted. Other surveys are not divided by season although they occur over multiple seasons. And the US West Coast is divided into two surveys that cover approx the same region and seasons, but in different years and possibly (?) with different gears.

This matters for model fitting details, particularly how we calculate a yearly mean biomass and whether we need different intercepts for each season. I/we need to think through this a bit more thoroughly to be sure we're not making a mistake.

**Comment by mpinsky** [comment]
Good point, though I'm a bit less concerned about this. We are defining separate surveys based on the way they are traditionally defined (NEUS has two surveys, West Coast had a triennial survey up until ~2004, then switched methods and now has an annual survey). I could possibly see combining the two NEUS surveys. I'll email Sean Lucey to see if the methods are substantially similar.

Other places this could be an issue are spring and fall for Newfoundland, but I know those two surveys are very different. I don't think we should combine them. 

Are there any others I'm missing?

**Comment by larogers123** [comment]
I agree to keeping surveys separate if they should be, but I think we need to adjust the model a bit. When I was testing the seasonal models, I was using NEUS, where survey and season were synonymous, essentially:

~ s(temp,by=survey) + s(rugosity) + s(meanbiomass) + survey,
were meanbiomass is the mean of the survey within a year. 

Thinking about a place like West Coast + Alaska, where surveys cover multiple seasons, and seasons could be sampled by multiple surveys, this should rather be:

~s(temp, by=season) + s(rugosity) + s(meanbiomass) + season + survey,
where again, meanbiomass is the mean of the survey within a year

I think we need the season factor in there if the temperature response is by season. However, there could be a case where season and survey are perfectly confounded, making problems for the model. I haven't tested this yet. Can do this afternoon.

Am I making it too complicated? Do you see a simpler way?

**Comment by mpinsky** [comment]
Ah, yes, I see what you mean. I agree that this looks right:
~s(temp, by=season) + s(rugosity) + s(meanbiomass) + season + survey

I also heard from Sean Lucey. Combining NEUS Spring and NEUS Fall into one "survey" with multiple seasons from our perspective seems fine. That will be one fewer survey level to estimate.
